miqi/execution/permission_engine.py

```
@@ -311,6 +311,19 @@ async def check(self, ctx: Any) -> PermissionDecision:
                 profile,
             )
 
+        # 6b. #646-v2 交互/确认类工具——用户通过卡片交互（非危险动作），
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)